src/libs/ReportUtils.ts

```diff
@@ -2941,6 +3000,68 @@ function getReportActionMessage(reportAction: ReportAction | EmptyObject, parent
     return reportAction?.message?.[0]?.text ?? '';
 }
 
+/**
+ * Get the title for an invoice room.
+ */
+function getInvoicesChatName(report: OnyxEntry<Report>): string {
+    const invoiceReceiver = report?.invoiceReceiver;
+    const isIndividual = invoiceReceiver?.type === CONST.INVOICE_RECEIVER_TYPE.INDIVIDUAL;
+    const invoiceReceiverAccountID = isIndividual ? invoiceReceiver.accountID : -1;
+    const policyID = isIndividual ? '' : invoiceReceiver?.policyID ?? '';
+    let isReceiver = false;
+
+    if (isIndividual && invoiceReceiverAccountID === currentUserAccountID) {
+        isReceiver = true;
+    }
+
+    if (!isIndividual && PolicyUtils.isPolicyMember(policyID, allPolicies)) {
+        isReceiver = true;
+    }
+
+    if (isReceiver) {
+        return getPolicyName(report, false, allPolicies?.[`${ONYXKEYS.COLLECTION.POLICY}${policyID}`]);
```

Comment:
```suggestion
        return getPolicyName(report);
```
If I'm a receiver, it's possible that the sender workspace is not in my list of workspaces, so we shouldn't rely on this data.